jobs/aws/eks-distro-build-tooling/aws-for-fluent-bit-helm-chart-postsubmits.yaml

```diff
@@ -14,7 +14,7 @@
 
 postsubmits:
   aws/eks-distro-build-tooling:
-  - name: aws-for-fluent-bit-helm-chart-postsubmits
+  - name: aws-for-fluent-bit-helm-chart-postsubmit
```

Comment:
Add "tooling" to the name

Reply:
eks-distro-build-tooling jobs don't have tooling in their names.

Reply:
Remove the label for image-build below too